src/main.cpp

```diff
@@ -290,9 +291,9 @@ void addConfigChain(EditProcessChain & procchain, Setting & configchain, Setting
             string modulename = chainelname;
             if(linkcfg.exists("module")) modulename = (const char *)linkcfg["module"];
             addChainLink(procchain, modulename, linkcfg);
-        } else if(chaincfgs.exists(chainelname)) {    // Check if it's the name of another chain
+        break; if(chaincfgs.exists(chainelname)) {    // Check if it's the name of another chain
```

Comment:
This logic is looping through each config section - breaking out after processing the first item will break the current logic.

`continue` might make sense here, however there are only 3 branches.